src/inc/consoletaeftemplates.hpp

```diff
@@ -270,81 +270,68 @@ namespace WEX::TestExecution
         static WEX::Common::NoThrowString ToString(const INPUT_RECORD& ir)
         {
             SetVerifyOutput verifySettings(VerifyOutputSettings::LogOnlyFailures);
-            WCHAR szBuf[1024];
-            VERIFY_SUCCEEDED(StringCchCopy(szBuf, ARRAYSIZE(szBuf), L"(ev: "));
+
+            WEX::Common::NoThrowString str;
+            str.Append(L"(ev: ");

```

Comment:
This change is not needed. I did it before I realized I had to upgrade the entire TAEF package to make it work.

Reply:
It's fine if you keep it in my opinion.